Require/index.js

```diff
@@ -211,4 +223,28 @@ describe("TNS require", function () {
         require('./RequireJsonCorruptFile');
         expect(TNSGetOutput()).toMatch(/JSON Parse error: Unterminated string$|Unexpected token/)
     });
+
+    it("shouldn't load invalid JSON file", function () {
+		require("./RequireJsonCorruptFile1");
+		expect(TNSGetOutput()).toMatch(/JSON Parse error: Unterminated string$|Unexpected token/)
+	});
+    
+    it("when using global in a module global should be defined", function () {
+		var module = require("./GlobalIsDefined/module");
```

Comment:
Couldn't you just return the global object in ```accessGlobalObject``` and check whether the returned object is != undefined ?